dask_searchcv/tests/test_model_selection.py

```diff
@@ -620,3 +621,39 @@ def test_scheduler_param_distributed(loop):
 def test_scheduler_param_bad():
     with pytest.raises(ValueError):
         _normalize_scheduler('threeding', 4)
+
+
+@pytest.mark.skipif(not _HAS_MULTIPLE_METRICS, reason="Added in 0.19.0")
+def test_multiple_metrics():
```

Comment:
Could also use a test with fit failures. There are a few examples in this file and `test_model_selection_sklearn.py`.

Also, if this test is borrowed from sklearn (or if there are tests using this feature that we could borrow from sklearn) then they should go in `test_model_selection_sklearn.py`.